### PR TITLE
Improve vs description

### PR DESCRIPTION
**Pull request by zhx828**
This includes the same code in pull request https://github.com/cBioPortal/cbioportal/pull/4189

No review needed if the pull request in portal is approved.

### CODE REVIEW

File: app/_components/cbio-util.js [hunk 1]
```diff
@@ -285,10 +285,10 @@ cbio.util = (function() {
 
     // check for IE 11
     if (!(browser.msie ||
-      browser.firefox ||
-      browser.chrome ||
-      browser.safari ||
-      browser.opera)) {
+        browser.firefox ||
```
Comment (kalletlak) [inline app/_components/cbio-util.js:288]: Is this file going to be updated in cbioportal or are these updates from cbioportal?

File: app/scripts/views/components/virtualStudy/virtualStudy.js [hunk 2]
```diff
@@ -102,9 +106,38 @@
...
+        };
+        var selectedStudies = {};
+        _.each(this.stats.studies, function(study) {
+          selectedStudies[study.id] = study.samples;
+          filteredOriginStudies.count += study.samples.length;
+        });
+        filteredOriginStudies.studies = this.stats.origin.map(function(studyId) {
+          var _studyData = iviz.datamanager.getStudyById(studyId);
+          var _count = 0;
+          if (_studyData.studyType === 'vs') {
```
Comment (kalletlak) [inline app/scripts/views/components/virtualStudy/virtualStudy.js]: if `_studyData` is `null/undefined`?

File: app/_components/cbio-util.js (hunk 1 above)
Comment (zhx828) [reply app/_components/cbio-util.js:288]: This is from portal.

File: app/scripts/views/components/virtualStudy/virtualStudy.js [hunk 3]
```diff
@@ -102,9 +106,38 @@
       enableShareCohortBtn: function(tooltip) {
         this.disableBtn(tooltip.find('.share-cohort'), false);
       },
-      getDefaultVSDescription: function() {
-        var def = new $.Deferred();
-        var selectedStudiesDisplayName = window.iviz.datamanager.getCancerStudyDisplayName(_.pluck(this.stats.studies, 'id'));
+      getFilteredOriginStudies: function() {
+        var selectedStudiesDisplayName = window.iviz.datamanager.getCancerStudyDisplayName(this.stats.origin);
```
Comment (kalletlak) [inline app/scripts/views/components/virtualStudy/virtualStudy.js:110]: does the `getCancerStudyDisplayName` function handles VS?

Comment (zhx828) [reply app/scripts/views/components/virtualStudy/virtualStudy.js:110]: It does.

Comment (kalletlak) [reply app/scripts/views/components/virtualStudy/virtualStudy.js:110]: I think it just handles saved VS but not shared VS

**Review by kalletlak — APPROVED** [review]

**Pull request merged**